client/app/components/PlayerHeadToHead.tsx

```diff
@@ -0,0 +1,208 @@
+        break;
+      case "rb":
+        setPositionBgColor(colors.rbBackground);
+        break;
+      case "wr":
+        setPositionBgColor(colors.wrBackground);
+        break; 
+      case "te":
+        setPositionBgColor(colors.teBackground);
+        break;
+      case "k":
+        setPositionBgColor(colors.kBackground);
+        break;
+      case "dst":
+        setPositionBgColor(colors.dstBackground);
+        break;
+    }
+  }, []);
+
+  return (
+    <View
+      flexDirection={"row"}
+      width={"100%"}
+      height={49}
+      backgroundColor={colors.secondaryBackground}
+
+      alignItems={"center"}
+      justifyContent={"space-around"}
+      position={"relative"}
+      borderRadius={5}
+      // boxShadow={"0px 4px 4px rgba(0, 0, 0, 0.25)"}
+      paddingLeft={10}
+      paddingRight={10}
+      marginTop={5}
+      marginBottom={5}
+    >
+
+{/**---------------------------------------------------------------- */}
+
+      <View
+        flexDirection={"column"}
+        alignItems={"flex-start"}
+
+        position="absolute" 
+        // Must use left instead of right margin
+        // Because want component's LHS to alwasy line up
+        left={8} 
+
+      >
+        <Text
+          color={colors.text}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.semiBold}
+          fontSize={14}
+        >
+          {props.player1NameShort}
+        </Text>
+        <Text
+          color={colors.text}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.normal}
+          fontSize={10}
+        >
+          {props.player1TeamCityShort} • {props.position.toUpperCase()}
+        </Text>
+      </View>
+
+{/**---------------------------------------------------------------- */}
+
+      <Text
+        color={colors.text}
+        textAlign='center'
+        fontFamily={typography.fonts.poppins.semiBold}
+        fontSize={17}
+        marginHorizontal={8}
+
+        position="absolute" 
+        // Must use right instead of left margin
+        // Because want this score's right hand side to always be 72.5 pixels from the center
+        right={WINDOW_WIDTH / 2 - 22.5 + 50} 
+
+      >
+        {props.player1Points}
+      </Text>
+
+{/**---------------------------------------------------------------- */}
+
+      <Box
+        bg={positionBgColor}
+        h={45}
+        w={45}
+        borderRadius={10}
+
+        position="absolute" // Set the position component's position to absolute
+        left={WINDOW_WIDTH / 2 - 22.5} // Calculate the left margin to center the position component
+        right={WINDOW_WIDTH / 2 - 22.5} // Calculate the right margin to center the position component    
+
+        display="flex"
+        justifyContent="center" 
+        alignItems="center" 
+      >
+        <Text
+          color={colors.textBlack}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.semiBold}
+          fontSize={20}
+        >
```

Comment:
I would add either add some margin top or some sort of alignment in order to have the position text perfectly centered.

Reply:
Oh you're right, they are not aligned in the center. I just went through and changed the line height to what it is in figma (30 px)